diff --git a/zlib-ng-2.1.5/CMakeLists.txt b/zlib-ng-2.1.5-mod/CMakeLists.txt
index 420a5c7..c97c496 100644
--- a/zlib-ng-2.1.5/CMakeLists.txt
+++ b/zlib-ng-2.1.5-mod/CMakeLists.txt
@@ -244,6 +244,8 @@ elseif(CMAKE_C_COMPILER_ID MATCHES "GNU" OR CMAKE_C_COMPILER_ID MATCHES "Clang")
         if(HAVE_NO_PEDANTIC_MS_FORMAT)
             list(APPEND WARNFLAGS_DISABLE -Wno-pedantic-ms-format)
         endif()
+        # Remove 'lib' prefix, the name zlib already has it.
+        set(CMAKE_SHARED_LIBRARY_PREFIX "")
     endif()
 endif()
 
